lib/basedriver/device-settings.js

```diff
@@ -25,6 +25,14 @@ const GLOBAL_DEFAULT_SETTINGS = {
   //      if a user use `width=750 × height=1334` pixels's base template image.
   fixImageTemplateScale: false,
 
+  // Users might have scaled template image to reduce their storage size.
```

Comment:
I would be nice to have this also described in some doc

Reply:
Yes. I'm thinking to add this in https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/image-elements.md